### PR TITLE
Docs cleanup from TIPs

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/tests/types/slot.rs [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
-        SlotIndex::new(10),
-        SlotCommitmentId::from_str(
-            "0x20e07a0ea344707d69a08b90be7ad14eec8326cf2b8b86c8ec23720fab8dcf8ec43a30e4a8cc3f1f",
-        )
-        .unwrap(),
-        RootsId::from_str("0xcf077d276686ba64c0404b9eb2d15556782113c5a1985f262b70f9964d3bbd7f").unwrap(),
-        5,
-        10,
+    // Test from https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#slot-commitment-id-1
+    let slot_commitment_json = r#"
```
Comment (DaughterOfMars) [inline sdk/tests/types/slot.rs]: Can we use the json macro pls

File: sdk/tests/types/slot.rs [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
+      }
+      "#;
+
+    let slot_commitment = serde_json::from_str::<SlotCommitment>(slot_commitment_json).unwrap();
+    let slot_commitment_bytes = slot_commitment.pack_to_vec();
+
+    assert_eq!(
+        slot_commitment_bytes,
+        [
+            3, 10, 0, 0, 0, 0, 0, 0, 0, 75, 2, 75, 62, 71, 40, 13, 5, 39, 42, 125, 19, 111, 12, 70, 78, 78, 19, 107,
```
Comment (DaughterOfMars) [inline sdk/tests/types/slot.rs]: Use the hex repr?

File: sdk/tests/types/slot.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: Sure

File: sdk/tests/types/slot.rs (hunk 2 above)
Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: I wouldn't do that tbh, it's like that in the TIP, I'm copy pasting it

File: sdk/tests/types/slot.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: Done

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]
Didnt check the moved tests, i assume was just small changes

**Review by DaughterOfMars — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**